fem/bilinearform_ext.cpp

```diff
@@ -100,6 +152,30 @@ void PABilinearFormExtension::Update()
       localX.SetSize(elem_restrict_lex->Height());
       localY.SetSize(elem_restrict_lex->Height());
    }
+
+   const int iFISz = a->GetFBFI()->Size();
+   if (int_face_restrict_lex && iFISz>0)
```

Comment:
Maybe `SetupRestrictionOperators()` can be reused here? Actually that will be done by `Assemble()` (which should be called after a call to this method). It seems we can  just reset the restriction pointers to `nullptr` here.

Reply:
Good suggestion.